src/components/overview-header/overview-header.js

```diff
@@ -38,37 +39,34 @@ class OverviewHeader extends React.PureComponent {
   }
 
   renderFooter() {
-    const { props } = this;
+    const { lightText, installLink, ghLink, contactLink } = this.props;
 
-    const btnClasses = classnames('btn txt-l round inline-block unprose mr24', {
-      'btn--white color-gray-dark': props.lightText,
-      'btn--blue': !props.lightText
+    const btnClasses = classnames('btn round unprose mr12 mb3', {
```

Comment:
💡 The `txt-l` felt really large, especially with the change to Assembly's default font size. Since this button is blue, I think it will still pop on the page.